.github/CODEOWNERS

```diff
@@ -59,20 +88,29 @@
 /core/domain/topic*.py @aks681
 /core/storage/topic/ @aks681
 /core/templates/dev/head/domain/topic/ @aks681
+/core/templates/dev/head/domain/topic_viewer @aks681
 /core/templates/dev/head/pages/topic_editor/ @aks681
 /core/templates/dev/head/pages/topic_viewer/ @aks681
 
 # Topics and skills dashboard project.
 /core/controllers/topics_and_skills_dashboard.py @aks681
+/core/templates/dev/head/domain/topics_and_skills_dashboard/ @aks681
 /core/templates/dev/head/pages/topics_and_skills_dashboard/ @aks681
 
 
 # Infrastructure.
 /core/storage/ @seanlip
+/core/platform/ @seanlip
 
 
 # Miscellaneous.
 /core/domain/html*.py @seanlip
+/core/templates/dev/head/pages/about/ @seanlip
+/core/templates/dev/head/pages/privacy/ @seanlip
```

Comment:
@seanlip, Added these to you as I thought these should go through admin review.

Reply:
SGTM